libraries/Preferences/src/Preferences.cpp

```diff
@@ -74,14 +78,20 @@ bool Preferences::clear(){
         log_e("nvs_erase_all fail: %s", nvs_error(err));
         return false;
     }
-    return true;
+    // return true;                                         // to undo changes: uncomment this line and...
+    err = nvs_commit(_handle);                              // ... delete the lines from from here...
+    if(err){
+        log_e("nvs_commit fail: %s", nvs_error(err));
+        return false;
+    }
+    return true;                                            // ... to here.
 }
 
 /*
  * Remove a key
  * */
 
-bool Preferences::remove(const char * key){
+bool Preferences::remove(const char * key){                 // modified to add an nvs_commit()
```

Comment:
same here, please remove trailing comments and the commented out return statement.